src/plugins/controls/public/control_group/editor/control_editor.tsx

```diff
@@ -107,8 +107,8 @@ export const ControlEditor = ({
     ) : null;
   };
 
-  const getTypeButtons = (controlTypes: string[]) => {
-    return controlTypes.map((type) => {
+  const getTypeButtons = () => {
+    return getControlTypes().map((type) => {
```

Comment:
No longer need to make a distinction for `isCreate` or not, so was able to simplify this quite a bit and simply **always** use `getControlTypes()` instead of dynamically deciding which controls to display via the parameter.